flask_exceptions/extension.py

```diff
@@ -15,7 +15,7 @@ def decorator(method):
         def wrapper(self, *args, **kwargs):
             """Innermost decorator wrapper - this is confusing."""
             if self.messages:
-                kwargs['message'] = args[0] if args else message
+                kwargs['message'] = args[0] if args else kwargs.get('message', message)
```

Comment:
It's a bit weird to change an arg to a kwarg, but in any case - this should also allow message to be passed as a kwarg. Without this change, a call `exceptions.bad_request(message='message')` would be overwritten with the default message.